api/api_controller.go

```diff
@@ -2297,3 +2300,23 @@ func (c *Controller) ImportFromS3InventoryHandler() repositories.ImportFromS3Inv
 		})
 	})
 }
+
+func (c *Controller) ConfigGetConfigHandler() configop.GetConfigHandler {
+	return configop.GetConfigHandlerFunc(func(params configop.GetConfigParams, user *models.User) middleware.Responder {
+		deps, err := c.setupRequest(user, params.HTTPRequest, []permissions.Permission{
+			{
+				// Should use repository creation permission but it is coupled to a repo id
+				Action:   permissions.ListRepositoriesAction,
```

Comment:
This action not particularly relevant to the operation, if this is the best we have then I would rather add a new action.  However that's *out of scope* for this PR, so instead I opened #764.
```suggestion
				// TODO(#764): Add a new action for reading configs?
				Action:   permissions.ListRepositoriesAction,
```